app/src/main/java/app/grapheneos/setupwizard/view/activity/LocationActivity.kt

```diff
@@ -23,6 +24,9 @@ class LocationActivity : SetupWizardActivity(
 
     override fun setupActions() {
         enabled.setOnClickListener { LocationActions.setEnabled(enabled.isChecked) }
+        findViewById<TextView>(R.id.enabled_desc).setOnClickListener {
```

Comment:
in this case, better to attach listener on the container view (the linear layout)?
and also provide visual indications of the click?

pushed the change here; you can rebase. https://github.com/jiteshsingh/platform_packages_apps_SetupWizard/commits/location_click_container/

Reply:
or I can merge my change later, either way.